### ISSUE TITLE
All twitteer proofs show as unreachable but posts remain

### ISSUE THREAD

**Issue by NukeManDan** [keybase/client#24397]
Every person I follow with a twitter proof shows as unreachable, and many I ask report the same.

The proofs are still on twitter and can be viewed from the link on someone's profile.

**Comment by haneefmubarak** [comment]
Can confirm the above. As an example, I'm @haneefmubarak on keybase with a proof at https://twitter.com/ihtsae/status/1118954420790673408?s=19 but the error is still thrown.

**Comment by haneefmubarak** [comment]
@mlsteele do you know if this is related to the changes in 6788497ca585ca674a56416a4554db599c01c860?

**Comment by mlsteele** [comment]
sure is, fix is on the way

**Comment by mlsteele** [comment]
Should be fixed now. We changed the endpoint used to fetch tweets and it returns links to https://keybase.io in a different way, as t.co links. There may be other types of incompatibilities so do let us know if there's more twitter proofs that were passing before that are still failing.

**Comment by haneefmubarak** [comment]
Confirmed resolved. Regular and `t.co` links are the only kind generated by the Twitter API, so recommend to close issue as resolved.

**Comment by bogosj** [comment]
I still don't see anyone (myself included) as showing a provable Twitter account if I browse on https://keybase.io/. Should I, or is there something that still needs to be rolled out?

**Comment by mlsteele** [comment]
@bogosj you're right, the client works now but the server (so website) still doesn't. Fix is on the way for that too

**Comment by mloskot** [comment]
Twitter proofs still - three years on - not working
https://github.com/keybase/client/issues/25759#issuecomment-1826795820

**Comment by StuxVT** [comment]
> sure is, fix is on the way

💀
